src/effects/VST/VSTEffect.h

```diff
@@ -605,6 +582,27 @@ class VSTEffectInstance final : public StatefulEffectBase::Instance,
    
    bool ProcessFinalize() noexcept override;
 
+
+   size_t SetBlockSize(size_t maxBlockSize) override;
+   size_t GetBlockSize() const override;
+
+   // this not?
+   //bool Process(EffectInstance& instance, EffectSettings& settings) override;
```

Comment:
There is nothing in the instance now corresponding to `Process`.  Instead you trust `PerTrackEffect` to do destructive effect processing in its `Process`, calling the `ProcessBlock` function of the instance.  And after other recent refactorings, that happens in member functions of an `EffectStage` object.

Reply:
ok